compiler/util/clangUtil.cpp

```diff
@@ -1869,6 +1870,8 @@ void makeBinaryLLVM(void) {
   output.keep();
   output.os().flush();
 
+  //finishClang is before the call to the debug finalize
+  deleteClang(info);
```

Comment:
Doesn't this call info->cgBuilder->Release() 3 times?
Does it work to set info->cgBuilder = NULL in the conditional in finishClang, so that info->cgBuilder->Release() is only called once?